config/initializers/action_mailer.rb

```diff
@@ -1,6 +1,12 @@
-# Read smtp config out of a config/smtp.yml file
-
-smtp_config = YAML::load(ERB.new(File.read(Rails.root.join('config', 'smtp.yml'))).result)
-if smtp_config.keys.include? Rails.env
-  ActionMailer::Base.smtp_settings = smtp_config[Rails.env].symbolize_keys
-end
\ No newline at end of file
+ActionMailer::Base.smtp_settings = {
+  address: ENV['SMTP_SERVER'] || "smtp.gmail.com",
+  port: ENV['SMTP_PORT'] || 587,
```

Comment:
I don't think so, previously we dumped the ENV variable (which always is a string) into the YAML file, `ActionMailer` probably handles the type casting.